src/main/java/step3/application/mainClass.java

```diff
@@ -0,0 +1,34 @@
+package step3.application;
+
+import step3.logic.CarCycleRunner;
+import step3.logic.CarMovement;
+import step3.logic.CarMovementLogger;
+import step3.view.input.InputScanner;
+import step3.view.input.InputView;
+import step3.view.output.OutputView;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class mainClass {
```

Comment:
클래스의 이름은 늘 대문자로 시작합니다. 
```suggestion
public class MainClass {